src/js/components/atoms/BaseLink/base-link.ts

```diff
@@ -7,6 +7,7 @@ import Events from 'constants/events';
 })
 
 export default class BaseLink extends Vue {
+    @Prop({ type: String, required: false }) type?: string;
```

Comment:
I know my old code but required default is false right? 😬 , I could also pick it up in my feature.